### PR TITLE
feat(pageContentWide gapsize): change page content wide gap size from…

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

- Change default gapSize value is from 10 to 20.

- Change VRT to reflect new changes.

### CODE REVIEW

**Review by tdeekens — APPROVED** [review]

File: visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx [hunk 1]
```diff
@@ -60,11 +60,11 @@ export const Component = () => (
           ),
         },
         {
-          name: 'two columns 1/1 with big gap',
-          path: 'two-columns-half-big-gap',
+          name: 'two columns 1/1 with small gap',
```
Comment (CarlosCortizasCT) [inline visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx:63]: There's another VRT called `two columns 2/1 with big gap` which we should update.
Also, I guess we should use the `l` size for the default two columns VRTs as now they have the big gap by default.

Comment (ddouglasz) [reply visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx:63]: Thank you. 

Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2967/commits/787277dd837da085601b9fc31c463bf44c9a3b84

File: visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx [hunk 2]
```diff
@@ -83,7 +83,7 @@ export const Component = () => (
           name: 'two columns 2/1 with big gap',
           path: 'two-columns-two-thirds-big-gap',
```
Comment (CarlosCortizasCT) [inline visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx:84]: The name of this test should also be changed
```suggestion
          name: 'two columns 2/1 with small gap',
          path: 'two-columns-two-thirds-small-gap',
```

Comment (ddouglasz) [reply visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx]: Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2967/commits/860a979efe49a26e94a9cfe903bc990e276dd91a

**Comment by CarlosCortizasCT** [comment]
Checking the failed VRTs I was wondering about the gap when using a single columns or a 1/1 columns.
With the change we applied, all of them now use the big gap (`64px`) by default. That means for single column, that every item will have a `64px` vertical gap:
image.

Also, for `1/1` columns, the gap will also be bigger:
image

@FilPob is this something we expect or should we have different defaults per wide layout?

**Comment by CarlosCortizasCT** [comment]
@ddouglasz, @FilPob I've been reviewing our previous decision to allow rendering all children in `PageContentWide` component when used in single column mode and it was because, when migrating views, the initial setup just swallowed extra children so developers where confused about what was happening, also because there was no warning about it.

I think it makes sense to only render the first children in this situation (`PageContentWide` in single mode) but also show a warning message (in browser console) in case more than one children is received so developers know about this behaviour.
Bear in mind when used in two columns mode, only the first two children are rendered.

I added a commit to implement this here: bdafc975dbc0a92eba574f14a26135b7fc062fc6

So, after all, we are changing the default gap size to be `64px` (instead of `32px`) for two columns layouts (in `PageContentWide` component); both `1/2 + 1/2` and `2/3 + 1/3`.
The gap size will only affect the horizontal spacing.

**Comment by CarlosCortizasCT** [comment]
We need to check the failing VRTs 🧐

**Review by emmenko — APPROVED** [review]
💯

**Comment by emmenko** [comment]
I left a comment in the VRTs regarding the button styles issue.

**Comment by CarlosCortizasCT** [comment]
> I left a comment in the VRTs regarding the button styles issue.

Yes, I recursively updated `emotion` dependencies and that solved the issues.

Thanks.

**Review by CarlosCortizasCT — APPROVED** [review]

**Pull request merged**